projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html

```diff
@@ -68,6 +71,45 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
     </dl>
   </section>
 
+  <!-- ISSUE DATA -->
+  <section *ngIf="record.metadata.type == 'issue'">
+    <header class="row">
+      <h3>{{ 'Issue data' | translate }}</h3>
+    </header>
+    <section class="mb-4">
+      <dl class="row mb-0">
+        <!-- numbering -->
+        <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Numbering</dt>
```

Comment:
Probably we can use template to avoid classes repetitions

Reply:
We can debate :-)
I think it's more understandable with this notation than if we used a template. It's also (for me) easier to update if we need to add a specific class (bold, color, ...) on a row/cell.